packages/clients/src/api/instance/v1/api.utils.ts

```diff
@@ -510,4 +511,23 @@ export class InstanceV1UtilsAPI extends API {
         ),
       )
       .then(res => ({ image: res.image }))
+
+  /**
+   * Sets the content of a user data on a server for the given key.
+   *
+   * @param request - The request {@link SetServerUserDataRequest}
+   */
+  setServerUserData = (request: Readonly<SetServerUserDataRequest>) =>
+    this.client.fetch<void>({
+      body: request.content,
+      headers: { 'Content-Type': 'text/plain' },
```

Comment:
This route takes a [`multipart/form-data`](https://developers.scaleway.com/en/products/instance/api/#patch-9ef3ec)

Reply:
(documentation seems outdated, call works)